README.md

```diff
@@ -101,6 +101,7 @@ Bearer auth plugin for Fastify
 - [`fastify-cookie`](https://github.com/fastify/fastify-cookie) Parse and set cookie headers
 - [`fastify-env`](https://github.com/fastify/fastify-env) Load and check configuration
 - [`fastify-formbody`](https://github.com/fastify/fastify-formbody)
+- [`fastify-graceful-shutdown`](https://github.com/hemerajs/fastify-graceful-shutdown)
```

Comment:
"Plugin to parse x-www-form-urlencoded bodies" may be wrong

Reply:
Oh... copy paste issue thanks